cmd/daprd/main.go

```diff
@@ -14,11 +14,16 @@ limitations under the License.
 package main
 
 import (
+	"fmt"
+	"os"
+
 	"go.uber.org/automaxprocs/maxprocs"
 
 	// Register all components
 	_ "github.com/dapr/dapr/cmd/daprd/components"
+	"github.com/dapr/dapr/cmd/daprd/options"
```

Comment:
Nit: Please move this one line below, so to leave a newline after the import of `components`